aten/src/ATen/native/cpu/UpSampleKernelAVXAntialias.h

```diff
@@ -94,91 +96,102 @@ void ImagingResampleHorizontalConvolution8u4x(
     uint32_t* C10_RESTRICT lineOut1,
     uint32_t* C10_RESTRICT lineOut2,
     uint32_t* C10_RESTRICT lineOut3,
+    int64_t out_xsize,
     const uint32_t* C10_RESTRICT lineIn0,
     const uint32_t* C10_RESTRICT lineIn1,
     const uint32_t* C10_RESTRICT lineIn2,
     const uint32_t* C10_RESTRICT lineIn3,
-    int xsize,
-    int* xbounds,
-    int16_t* kk,
+    const int64_t* idx_ptr_xmin,
+    const int64_t* idx_ptr_size,
+    const int16_t* kk,
     int kmax,
-    int coefs_precision);
+    unsigned int coefs_precision);
 
 void ImagingResampleHorizontalConvolution8u(
     uint32_t* C10_RESTRICT lineOut,
+    int64_t out_xsize,
     const uint32_t* C10_RESTRICT lineIn,
-    int xsize,
-    int* xbounds,
-    int16_t* kk,
+    const int64_t* idx_ptr_xmin,
+    const int64_t* idx_ptr_size,
+    const int16_t* kk,
     int kmax,
-    int coefs_precision);
+    unsigned int coefs_precision);
 
 void ImagingResampleVerticalConvolution8u(
     uint32_t* C10_RESTRICT lineOut,
-    const uint32_t* C10_RESTRICT imIn,
-    int xmin,
-    int xmax,
-    int16_t* k,
-    int coefs_precision,
-    int xin);
+    const uint32_t* C10_RESTRICT lineIn,
+    int64_t xsize,
+    int64_t ids_min,
+    int64_t ids_size,
```

Comment:
what does `ids` stand for?

Reply:
ids is for index

Reply:
Why not `idx`?

Reply:
`ids_size` and `ids_min` I previously used in https://github.com/pytorch/pytorch/blob/f3db2a634103690c6a87f5f34d29cd6c8b622a92/aten/src/ATen/native/cpu/UpSampleKernel.cpp#L195-L198

Term `index` is coming from interpolation algorithm where we were precomputing input data indices as input was restrided for tensor iterator. I agree that previously I rather freely used `ids` and `wts` to indicate an index and a weigth value. 

`ids_min` and `ids_size` technically mean that for a given interpolated point we provide indices that start from some `min` value up to `min + size`.

Reply:
if it's consistent with the rest of the code then I guess that's okay-ish. It's still very surprising to me because I've never seen `ids` as a shortening of index (indices?) before.